lib/orbit/operator_sign_in.ex

```diff
@@ -12,6 +12,7 @@ defmodule Orbit.OperatorSignIn do
           signed_in_by_user: User.t(),
           signed_in_at: DateTime.t(),
           rail_line: RailLine.t(),
+          radio_number: integer(),
```

Comment:
This should probably be nillable (for records that are already in the database, and for any records between merging this PR and #164 )